jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -31,6 +36,27 @@ declare -r PARAM_VARS_OPTIONAL=(
   'TERMINATION_LOG_PATH'
 )
 
+unset FINAL_RESULT
+
+# terminate sets the passed RC and exits
+# terminate without parameter exits with RC 0
```

Comment:
I would not imply "success" when an explicit parameter is missing, because it is less understandable when reading `terminate`.

Reply:
Changed